py-polars/src/series.rs

```diff
@@ -1379,17 +1379,50 @@ macro_rules! impl_ufuncs {
                 let (out_array, av) =
                     unsafe { aligned_array::<<$type as PolarsNumericType>::Native>(py, size) };
 
-                debug_assert_eq!(out_array.get_refcnt(), 1);
+                debug_assert_eq!(
+                    {
+                        let refcnt = out_array.get_refcnt();
```

Comment:
Shall we make a `get_refcnt` function to have this logic in one place.

Reply:
Probably a better idea, yeah

Reply:
where should this function be put?

Reply:
Let's go for `utils.rs` for now. Later we can create a python module where we keep this kind of stuff.

Reply:
I put it in `npy.rs` as for now get_refcnt only supports `PyArray1` types.